app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeViewModel.kt

```diff
@@ -354,7 +357,14 @@ class AttendeeViewModel(
                     Timber.d("Failed charging the user")
                 }
             }, {
-                mutableMessage.value = resource.getString(R.string.payment_not_complete_message)
+                if (it is HttpException) {
+                    val body = it.response()?.errorBody()
+                    val jsonObject = JSONObject(body?.string()).getJSONArray(ERRORS)[0]
```

Comment:
This will crash the app if not present

Reply:
So I think, try-catch can be used.

Reply:
Copy ErrorUtils from orga app and use getErrorDetails, so this can be handled once for everywhere

Reply:
Updated